#--- src/c-interface-gui.cc	2009-05-26 01:30:03.000000000 +0200
#+++ src/c-interface-gui.cc.new	2009-06-03 11:14:25.000000000 +0200
#@@ -74,6 +74,7 @@
# #include "cmtz-interface.hh"
# #include "mmdb.h"  // for centre of molecule
# #include "clipper/core/clipper_instance.h"
#+#include <gtk/gtk.h>
# 
# #if (GTK_MAJOR_VERSION > 2 || (GTK_MAJOR_VERSION == 2 && GTK_MINOR_VERSION > 5))
# #define HAVE_GTK_COMBO_BOX_GET_ACTIVE_TEXT
--- src/c-interface-gui.cc	2009-06-03 11:55:35.000000000 +0200
+++ src/c-interface-gui.cc.new	2009-06-03 13:12:19.000000000 +0200
@@ -5355,11 +5355,12 @@
 #if (GTK_MAJOR_VERSION > 1)
 #if (GTK_MAJOR_VERSION > 2) || (GTK_MINOR_VERSION > 14) 
    for (int i=0; i<5; i++)
-      gtk_scale_add_mark(GTK_RANGE(h_scale),
+      gtk_scale_add_mark(GTK_SCALE(h_scale),
 			 float (i-2) * 0.5 * sharpening_limit,
-			 GTK_POS_BOTTOM);
-   gtk_scale_add_mark(GTK_RANGE(h_scale), -sharpening_limit, GTK_POS_BOTTOM, "Blur");
-   gtk_scale_add_mark(GTK_RANGE(h_scale),  sharpening_limit, GTK_POS_BOTTOM, "Sharp");
+			 GTK_POS_BOTTOM,
+			 "");
+   gtk_scale_add_mark(GTK_SCALE(h_scale), -sharpening_limit, GTK_POS_BOTTOM, "Blur");
+   gtk_scale_add_mark(GTK_SCALE(h_scale),  sharpening_limit, GTK_POS_BOTTOM, "Sharp");
 #endif   
 #endif

